src/components/Regist/Regist.js

```diff
@@ -0,0 +1,221 @@
+
+import style from '../Loginn/Login.module.css';
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {Routes , Route , Link} from 'react-router-dom'
+
+function Regist() {
+  const [userName , setUserName] = useState('')
+  const [email , setEmail] = useState('')
+  const [pass , setPass] = useState('')
+  const [confirmPass , setConfirmPass] = useState('')
+
+  const [userNameDirty , setUserNameDirty] = useState(false)
+  const [emailDirty , setEmailDirty] = useState(false)
+  const [passDirty , setPassDirty] = useState(false)
+  const [confPassDirty , setConfPassDirty] = useState(false)
+
+  const  [userError , setUserError] = useState('Ім\'я не може бути пустим')
+  const [emailEror , setEmailEror] = useState('Емеїл не може бути пустим')
+  const [passEror , setPassEror] = useState('Пароль не може бути пустим')
```

Comment:
You can't use a shortened version of a word if it means something else. For example, "password" can't be shortened to "pass" because "pass" means "to skip," which creates misunderstandings.